sdk/core/core-tracing/src/createSpan.ts

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Span, SpanOptions, SpanKind } from "@opentelemetry/api";
+import { getTracer } from "../src/tracerProxy";
+
+/**
+ * Tracing options to set on an operation.
+ * @hidden
+ */
+export interface OperationTracingOptionsLike {
+  /**
+   * OpenTelemetry SpanOptions used to create a span when tracing is enabled.
+   */
+  spanOptions?: SpanOptions;
+}
+
+/**
+ * Configuration for creating a new Tracing Span
+ * @hidden
+ */
+export interface SpanConfig {
```

Comment:
marked as hidden, but still shows up in d.ts because it's used by `createSpanFunction` ?

Reply:
(answered here: https://github.com/Azure/azure-sdk-for-js/pull/13887/files/7092a579934657a4e1adf496caecf29f0715f475#r580646790)

Basically, was going for "internal detail, but exported because it's used by other modules"